pkg/cmd/flags.go

```diff
@@ -79,11 +79,6 @@ func AutocompleteByokKeyIds(client *ccloudv2.Client) []string {
 	return suggestions
 }
 
-func AddByokProviderFlag(cmd *cobra.Command) {
```

Comment:
Nice! I think this was originally a separate function because they didn't support all three. Now that they do, we can definitely delete this.